Marlin/Configuration.h

```diff
@@ -2564,23 +2564,28 @@
 //#define DWIN_CREALITY_LCD
 
 //
-// ADS7843/XPT2046 ADC Touchscreen such as ILI9341 2.8
+// Touch Screen Settings(Resistance screen or Capacitance screen)
 //
 //#define TOUCH_SCREEN
 #if ENABLED(TOUCH_SCREEN)
+  #define RES_TOUCH_SCREEN // ADS7843/XPT2046 ADC Touchscreen such as ILI9341 2.8
+  //#define CAP_TOUCH_SCREEN // GT911 Capacitive touch screen such as BIQU_BX_TFT70
```

Comment:
Thinking here, should not the RES/CAP driver type be linked with the TFT defines?
Here: https://github.com/MarlinFirmware/Marlin/blob/76ec7040f1209ea9e0f0132782be62f3054601bb/Marlin/src/inc/Conditionals_LCD.h#L1126

Each TFT should define if it's `RES_TOUCH_SCREEN` or `CAP_TOUCH_SCREEN` and we don't need expose this config to the user.